backend_new/src/services/user.service.ts

```diff
@@ -4,6 +4,7 @@ import {
   NotFoundException,
   UnauthorizedException,
 } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
```

Comment:
I'm not sure I like using the config service here. In the current backend we use the config service to sorta hardcode some env variables in. 

I think my preference is to just use process.env and put defaults like that into the template.

Hopefully that makes some sense, but it saves us a package I don't think we really need, and may may it easier to understand what exactly needs to be in the .env file for the app to work

Reply:
I think it could be useful to setup the ConfigModule so that there is a validationSchema against it. That way we don't have to rely on the template file and there are failures on startup if we don't have them all in the file. Similar to how we are using it shared.module.ts in the old backend. Thoughts?

Reply:
An issue has been created to go back and update all places to use a config service